docs/vta/install.md

```diff
@@ -234,7 +233,38 @@ export XILINX_VIVADO=${XILINX_PATH}/Vivado/2019.1
 export PATH=${XILINX_VIVADO}/bin:${PATH}
 ```
 
-### Custom VTA Bitstream Compilation
+### Intel Toolchain Installation
+
+It is recommended to use `Intel Quartus Prime 18.1`, since the test scripts contained in this document have been tested on this version. 
+
+You would need to install Intel's FPGA compilation toolchain, [Quartus Prime Lite](http://fpgasoftware.intel.com/?edition=lite), which is a license-free version of the Intel Quartus Prime software.
+
+#### Obtaining and Launching the Quartus GUI Installer
+
+1. Go to the [download center](http://fpgasoftware.intel.com/?edition=lite), and download the linux version of `Quartus Prime (include Nios II EDS)` and `Cyclone V device support` files in the `Separate file` tab. This avoid downloading unused device support files.
+2. Sign in the form if you have an account, or register on the right side of the web page to create an account.
+3. After signed in, you are able to download the installer and the device support files.
+4. Now that the files are downloaded, go to your `Downloads` directory, and change the file permissions:
+```bash
+chmod u+x QuartusLiteSetup-18.1.0.625-linux.run
+```
+5. Now ensure both the installer and device support files are in the same directory, and you can run the install with:
+```bash
+./QuartusLiteSetup-18.1.0.625-linux.run
+```
+6. Follow the instructions on the pop-up GUI form, and install all the content in the `/usr/local` directory. After installation, `/usr/local/intelFPGA_lite/18.1` would be created and the Quartus program along with other programs would be available in the folder.
+
+#### Environment Setup
+
+Similar to what should be done for Xilinx toolchain, the following line should be added to your `~/.bashrc`.
+```bash
+# Intel Quartus 18.1 environment
+export QUARTUS_ROOTDIR="/usr/local/intelFPGA_lite/18.1/quartus"
```

Comment:
We're missing the QSYS path here, and the build will break. 

Add the following lines so we have a path to `qsys-script` in 18.1
```
export PATH=${INTEL_QUARTUS}/sopc_builder/bin:${PATH}
```

Reply:
Nice catch. It's really important to notify the users to include the path.